kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/utilities/qaseIoUtils/QaseExtension.java

```diff
@@ -35,21 +35,21 @@ public class QaseExtension implements TestExecutionListener {
     private final ResultsApi resultsApi = new ResultsApi(apiClient);
     private final Map<TestIdentifier, Long> testStartTimes = new ConcurrentHashMap<>();
     private static final String QASE_PROJECT = "KAFKAUI";
-    private static final String QASE_ENABLE = "true";
 
     static {
         String qaseApiToken = System.getProperty("QASEIO_API_TOKEN");
-        if (qaseApiToken == null || StringUtils.isEmpty(qaseApiToken)) {
-            throw new RuntimeException("QaseIO API token should be present");
-        }
-        if ("true".equalsIgnoreCase(System.getProperty("QASEIO_CREATE_TESTRUN"))) {
-            System.setProperty("QASE_RUN_NAME", "Automation run " +
-                    new SimpleDateFormat("dd/MM/yyyy HH:mm:ss").format(new Date()));
+        if (StringUtils.isEmpty(qaseApiToken)) {
+           System.setProperty("QASE_ENABLE", "false");
```

Comment:
can we add some logging here with warn level please? so the folks who run this locally won't get frustrated